mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,43 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+     b has not been defined
   2. Remove the line that throws the error.
   3. What is printed to the console?
+    2, 6, 4, 9, 6, 13, 8
   4. How many times is "f1" called?
+    Two times
   5. How many times is "f2" called?
+    Three times
   6. What value does the "a" parameter take in the first "f1" call?
+    8
```

Comment:
this exercise is very unclear because it reuses the same variable names. If you had a look at the solutions you will see this answer is 1 because the `a` the question is referring to is the parameter in the function definition for `f1`, i.e. this `a`
```
const f1 = function (a, b) {
```